tabbed-mode-explainer.md

```diff
@@ -0,0 +1,65 @@
+# Tabbed Mode
+
+Author: Louise Brett (loubrett@google.com)
+
+## Overview
+
+Currently PWAs in a standalone window can only have one page open at a time. Some apps expect users to have many pages open at once. Tabbed mode adds a tab strip to standalone web apps that allows multiple tabs (within the app's scope) to be open at once.
+
+This document describes a new display mode `tabbed` and a new manifest member `tab_strip` which lets apps enable the tab strip and customize it.
+
+## Proposal
+
+Add a new display override option `tabbed` which behaves similarly to `standalone`, but with an added tab strip.
+
+```json
+"display_override": ["tabbed"]
+```
+
+Add a new manifest field `tab_strip` which allows apps to customize the tab strip. The `tab_strip` field will only be used when the display mode is `tabbed`.
+
+```
+"tab_strip": {
+    "home_tab": "auto" | "absent" | {
+        "url": "auto" | <url>,
+        "icons": "auto" | [...],
+    },
+    "new_tab_button": "auto" | "absent" | {
+        "url": "auto" | <url>,
+        "icons": "auto" | [...],
+    },
+},
+```
+
+The home tab is a pinned tab that, if enabled for an app, should always be present when the app is open. This tab should never navigate - links clicked from this tab should open in a new app tab. Apps can choose to customize the URL this tab is locked to and the icon displayed on the tab.
+
+The new tab button, if present, should open a new tab at a URL that is within the scope of the app. The app may choose to set a custom URL and icon for this button.
+
+If the `tab_strip` field is not present, the particular subfields' `auto` values should be used. The user agent can decide what values to use for `auto`. For example it might set `url` to be the `start_url`.
```

Comment:
```suggestion
If the `tab_strip` field is not present, the particular subfields' `auto` values should be used. The user agent can decide what values to use for `auto`. For example it might set `new_tab_button.url` to be the `start_url`.
```

(Suggestion from https://github.com/WICG/manifest-incubations/pull/55/files#r875472212.)